lib/workers/repository/index.ts

```diff
@@ -31,9 +31,9 @@ export async function renovateRepository(
   setMeta({ repository: config.repository });
   logger.info({ renovateVersion: pkg.version }, 'Repository started');
   logger.trace({ config });
-  let repoResult: ProcessResult;
+  let repoResult: ProcessResult | undefined = undefined;
```

Comment:
```suggestion
  let repoResult: ProcessResult | undefined;
```
no need for explicit undefined

Reply:
Leads to `Variable 'repoResult' is used before being assigned`

Reply:
🤦‍♂️ ok, can we use null only?

Reply:
Ah, sorry, it's actually okay